test/driver/exception_handling/run.t

```diff
@@ -77,47 +79,52 @@ can be reported about it.
   $ echo "let x = 1+1. " > impl.ml
   $ echo "let _ = [%gen_raise_located_error]" >> impl.ml
   $ ./extender.exe -embed-errors impl.ml
-  [%%ocaml.error "A raised located error"]
+  [%%ocaml.error "(context-free ppx pass) A raised located error"]
 
  In the case of derivers
 
   $ echo "let x = 1+1. " > impl.ml
   $ echo "type a = int" > impl.ml
   $ echo "type b = int [@@deriving deriver_located_error]" >> impl.ml
   $ ./deriver.exe -embed-errors impl.ml
-  [%%ocaml.error "A raised located error"]
+  [%%ocaml.error "(context-free ppx pass) A raised located error"]
 
  In the case of whole file transformations:
 
   $ echo "let x = 1+1. " > impl.ml
   $ ./whole_file_located_error.exe -embed-errors impl.ml
-  [%%ocaml.error "A located error in a whole file transform"]
+  [%%ocaml.error
+    "The following located exception was raised during the whole file transformation phase of the ppx \"raise_exc\":\nA located error in a whole file transform\nThe use of located exception is discouraged in whole file transformation as it prevents other errors to be reported. Instead, errors should be embedded in the AST in extension nodes. You might want to file an issue to the ppx authors."]
 
 3. Raising an exception. The exception is not caught by the driver.
```

Comment:
You could update this test now by saying that it's caught and it's re-raised with an improved message.

Reply:
How many times will I forget to update the text when the tests change behavior? :sweat_smile: